ctest-next/tests/input/macro.h

```diff
@@ -0,0 +1,13 @@
+#include <stdint.h>
+
+struct VecU8
+{
+    uint8_t x;
+    uint8_t y;
+};
+
+struct VecU16
+{
+    uint16_t x;
+    uint16_t y;
+};
```

Comment:
Please double check all files for a trailing \n, there are a few still (sometimes messes with tooling otherwise)